### PR TITLE
\ e & \equiv & \hyperlink{h T}{h(T)}\\

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -1661,6 +1661,7 @@ \section{Signing Transactions}\label{app:signing}
 We assume the existence of functions $\mathtt{\small ECDSAPUBKEY}$, $\mathtt{\small ECDSASIGN}$ and $\mathtt{\small ECDSARECOVER}$. These are formally defined in the literature, \eg by \cite{ECDSAcerticom}.
 \begin{eqnarray}
 \mathtt{\small ECDSAPUBKEY}(p_{\mathrm{r}} \in \mathbb{B}_{32}) & \equiv & p_{\mathrm{u}} \in \mathbb{B}_{64} \\
+\ e & \equiv & \hyperlink{h T}{h(T)}\\
```
Comment (pirapira) [inline Paper.tex]: Where is this newly defined variable `e` used?

Comment (jamesray1) [reply Paper.tex]: For clarity, but the label needs updating as it has a space.

Comment (jamesray1) [reply Paper.tex]: It is defined because it is used as an input to ECDSASIGN and ECDSARECOVER.

File: Paper.tex [hunk 2]
```diff
@@ -1661,6 +1661,7 @@ \section{Signing Transactions}\label{app:signing}
 We assume the existence of functions $\mathtt{\small ECDSAPUBKEY}$, $\mathtt{\small ECDSASIGN}$ and $\mathtt{\small ECDSARECOVER}$. These are formally defined in the literature, \eg by \cite{ECDSAcerticom}.
 \begin{eqnarray}
 \mathtt{\small ECDSAPUBKEY}(p_{\mathrm{r}} \in \mathbb{B}_{32}) & \equiv & p_{\mathrm{u}} \in \mathbb{B}_{64} \\
+\ e & \equiv & \hyperlink{h_of_T}{h(T)}\\
```
Comment (pirapira) [inline Paper.tex]: I think `e` should be explained in the next paragraph (which explains p_r, p_u and v).

Comment (jamesray1) [reply Paper.tex]: Done.

**Comment by pirapira** [comment]
What about
```
$e$ is the hash of the transaction $\hyperlink{h_of_T}{h(T)}$
```
?

**Comment by jamesray1** [comment]
OK.

**Comment by pirapira** [comment]
There was a non-break space http://www.fileformat.info/info/unicode/char/00a0/index.htm instead of a space.

**Comment by jamesray1** [comment]
Hmm, I wonder how that happened.

**Pull request merged**